src/component/vocabulary/VocabularySummary.tsx

```diff
@@ -63,6 +70,13 @@ export class VocabularySummary extends EditableComponent<VocabularySummaryProps>
         });
     };
 
+    public onRemove = () => {
+        const onCloseRemove = this.onCloseRemove;
```

Comment:
IMHO, it is not necessary to use a temporary variable for callback when using fat arrow function.